tests/clientEncryption/clientEncryption-encryptExpression-001.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+MongoDB\Driver\ClientEncryption::encryptExpression() encrypts $and operator
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongocrypt(); ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_server_version('<', '7.0'); ?>
+<?php skip_if_not_clean(CSFLE_KEY_VAULT_DATABASE_NAME, CSFLE_KEY_VAULT_COLLECTION_NAME);
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = create_test_manager();
+
+$clientEncryption = $manager->createClientEncryption([
+    'keyVaultNamespace' => CSFLE_KEY_VAULT_NS,
+    'kmsProviders' => ['local' => ['key' => new MongoDB\BSON\Binary(CSFLE_LOCAL_KEY, 0)]],
+]);
+
+$keyId = $clientEncryption->createDataKey('local');
+
+$encryptOpts = [
+    'keyId' => $keyId,
+    'algorithm' => MongoDB\Driver\ClientEncryption::ALGORITHM_RANGE_PREVIEW,
+    'queryType' => MongoDB\Driver\ClientEncryption::QUERY_TYPE_RANGE_PREVIEW,
+    'contentionFactor' => 0,
+    'rangeOpts' => ['min' => 0, 'max' => 200, 'sparsity' => 1],
+];
+
+$expr = [
+    '$and' => [
+        ['encryptedInt' => ['$gte' => 6]],
+        ['encryptedInt' => ['$lte' => 200]],
+    ],
+];
+
+$encryptedExpr = $clientEncryption->encryptExpression($expr, $encryptOpts);
+
+var_dump($encryptedExpr->{'$and'}[0]->encryptedInt->{'$gte'} instanceof MongoDB\BSON\Binary);
+var_dump($encryptedExpr->{'$and'}[0]->encryptedInt->{'$gte'}->getType() === MongoDB\BSON\Binary::TYPE_ENCRYPTED);
+
+var_dump($encryptedExpr->{'$and'}[1]->encryptedInt->{'$lte'} instanceof MongoDB\BSON\Binary);
+var_dump($encryptedExpr->{'$and'}[1]->encryptedInt->{'$lte'}->getType() === MongoDB\BSON\Binary::TYPE_ENCRYPTED);
```

Comment:
Note: dumping the entire `$encryptedExpr` object and asserting the contents of both Binary objects with `%a` (see: [EXPECTF](https://qa.php.net/phpt_details.php#expectf_section)) was unreliable for the same reasons as Session assertions. See https://github.com/mongodb/mongo-php-driver/commit/2104d6dff3a81fd79b00d00ce03eaf272501a05f for more context.